bom/pom.xml

```diff
@@ -24,7 +24,7 @@
         <!-- compile/runtime dependency versions, managed in the BOM -->
         <classmateVersion>1.0.0</classmateVersion>
         <ehcacheVersion>2.6.9</ehcacheVersion>
-        <infinispanVersion>8.0.2.Final</infinispanVersion>
+        <infinispanVersion>8.2.0.Final</infinispanVersion>
```

Comment:
Yes, the micro difference for Narayana should be fine.

Regarding ISPN, let's have a chat.